cluster/transport/client.go

```diff
@@ -65,10 +65,11 @@ type Client struct {
 	connLock   sync.Mutex
 	leaderAddr string
 	leaderConn *grpc.ClientConn
+	rpcMaxSize int
 }
 
-func NewClient(r rpcAddressResolver) *Client {
-	return &Client{rpc: r}
+func NewClient(r rpcAddressResolver, rpcMaxSize int) *Client {
+	return &Client{rpc: r, rpcMaxSize: rpcMaxSize}
```

Comment:
Should I do something different here so the original function signature is preserved or is it fine as long I've replaced all usages in the weaviate codebase?